multinet/uploaders/swagger/d3_json.yaml

```diff
@@ -0,0 +1,27 @@
+Upload a D3 JSON graph file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw JSON text
+    schema:
+      type: string
+
+responses:
+  200:
+    description: D3 data uploaded to tables
+    schema:
+      type: object
+      properties:
+        edgecount:
+          type: integer
+        nodecount:
+          type: integer
+
```

Comment:
Can return 400 if the validation fails

Reply:
8d600b7fd656afab80ac9233b661e207677d07e4